alpha/apps/kaltura/lib/rules/conditions/kCondition.php

```diff
@@ -115,4 +121,29 @@ public function shouldDisableCache($scope)
 	{
 		return true;
 	}
+
+	public function getExtraProperties()
```

Comment:
Better create an interface for this use, getExtraProperties method is implemented in multiple classes.